pkg/appliance/backup.go

```diff
@@ -213,11 +213,19 @@ func PerformBackup(cmd *cobra.Command, args []string, opts *BackupOpts) (map[str
 	retryStatus := func(ctx context.Context, applianceID, backupID string, tracker *tui.Tracker) error {
 		bo := backoff.NewExponentialBackOff()
 		bo.MaxElapsedTime = 0
+		networkErrors := 0
 		return backoff.Retry(func() error {
 			status, err := backupAPI.Status(ctx, applianceID, backupID)
 			if err != nil {
+				if errors.Is(err, context.DeadlineExceeded) {
+					networkErrors++
+					if networkErrors >= 5 {
+						return backoff.Permanent(err)
```

Comment:
I think it would be suitable to have a human friendly error message here if this happends, since context deadline error is not very user friendly.

Reply:
Wouldn't this snippet in root.go do just that though? https://github.com/appgate/sdpctl/blob/c1b18d676c320cfa179acf44810270f612167aff/cmd/root.go#L211-L213

Reply:
no, since this err is handled within the command, https://github.com/appgate/sdpctl/blob/3d4972f1a876fe36beb6b1cb9fabec8d522f3dad/pkg/appliance/backup.go#L293-L295

Reply:
Fixed in 4f5d038f3fc5a36961098b9a19f97267bd95c15c